wallet/src/main/scala/org/bitcoins/wallet/models/SpendingInfoTable.scala

```diff
@@ -166,8 +166,7 @@ sealed trait SpendingInfoDb
   * it is spent, how many (if any) confirmations it has
   * or which block/transaction it was included in.
   *
-  * That is rather handled by
-  * [[org.bitcoins.wallet.models.WalletTXOTable WalletTXOTable]].
+  * That is rather handled by WalletTXOTable.
```

Comment:
This is my fault, I forgot to update the Scaladoc a while ago. Could you change the doc for this class to

```
This table stores the necessary information to spend
a transaction output (TXO) at a later point in time. It 
also stores how many confirmations it has, whether 
or not it is spent (i.e. if it is a UTXO or not) and the 
TXID of the transaction that created this output. 
```